### PR TITLE
Update VTEX - Catalog API.json

### PR DESCRIPTION
**Pull request by kriegerderraik**
Get Subcollection's SKUs

### CODE REVIEW

File: VTEX - Catalog API.json [hunk 1]
```diff
@@ -13524,6 +13524,94 @@
...
+                        "schema": {
+                            "type": "integer",
+                            "default": 1
+                        }
+                    },
+                    {
+                        "name": "size",
+                        "in": "query",
+                        "required": true,
+                        "description": "Size of the itens in a page",
```
Comment (georgebrindeiro) [inline VTEX - Catalog API.json:13601]: Number of items in a page

File: VTEX - Catalog API.json [hunk 2]
```diff
@@ -13524,6 +13524,94 @@
...
+                        "schema": {
+                            "type": "integer",
+                            "default": 1
```
Comment (georgebrindeiro) [inline VTEX - Catalog API.json:13604]: Better default would be 50

File: VTEX - Catalog API.json [hunk 3]
```diff
@@ -13529,7 +13529,7 @@
                 "tags": [
                     "Subcollection"
                 ],
-                "summary": "Get Subcollection's SKU ",
+                "summary": "Get Subcollection's SKUs",
```
Comment (georgebrindeiro) [inline VTEX - Catalog API.json:13532]: LGTM

File: VTEX - Catalog API.json [hunk 4]
```diff
@@ -13598,7 +13598,7 @@
                         "name": "size",
                         "in": "query",
                         "required": true,
-                        "description": "Size of the itens in a page",
+                        "description": "Quantity of the itens in a page",
```
Comment (georgebrindeiro) [inline VTEX - Catalog API.json:13601]: Still prefer "Number of items in a page"

(note that "itens" is misspelled in this commit)

File: VTEX - Catalog API.json (hunk 1 above)
Comment (kriegerderraik) [reply VTEX - Catalog API.json]: I was replacing it for "quantity". Do you think "number" is a better option?

File: VTEX - Catalog API.json (hunk 2 above)
Comment (kriegerderraik) [reply VTEX - Catalog API.json:13604]: Roger

File: VTEX - Catalog API.json (hunk 4 above)
Comment (kriegerderraik) [reply VTEX - Catalog API.json:13601]: Roger!

**Pull request merged**